### PR TITLE
Adding React dependency

### PR DESCRIPTION
**Pull request by MohammadSommakia**

### CODE REVIEW

**Review by NourSammour — CHANGES_REQUESTED** [review]
node_modules should be ignored, please add it to .gitignore file
or rebase your fork, @AbdilFattahBoshi already did that
https://github.com/NourSammour/biletiniz/pull/12

**Comment by MohammadSommakia** [comment]
Sorry i forgot to put it in ignore file and  i create it
 without reading that Abdil Fattah did that.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
```
Comment (NourSammour) [inline package-lock.json]: while we are using `yarn` we should not use `package-lock`
please remove this file

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Review by NourSammour — CHANGES_REQUESTED** [review]
again i see node_modules files, please ignore it

**Pull request closed without merging**